File: app/src/main/java/code/name/monkey/retromusic/ui/fragments/base/AbsLibraryPagerFragment.java
package code.name.monkey.retromusic.ui.fragments.base;

import android.os.Bundle;

import code.name.monkey.retromusic.ui.fragments.mainactivity.LibraryFragment;

public class AbsLibraryPagerFragment extends AbsMusicServiceFragment {


    public LibraryFragment getLibraryFragment() {
        return (LibraryFragment) getParentFragment();
    }

    @Override
    public void onActivityCreated(Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        setHasOptionsMenu(true);
    }
}

File: app/src/main/java/code/name/monkey/retromusic/ui/fragments/base/AbsLibraryPagerFragment.kt
package code.name.monkey.retromusic.ui.fragments.base

import android.os.Bundle
import code.name.monkey.retromusic.ui.fragments.mainactivity.LibraryFragment

open class AbsLibraryPagerFragment : AbsMusicServiceFragment() {


    val libraryFragment: LibraryFragment
        get() = parentFragment as LibraryFragment

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        setHasOptionsMenu(true)
    }
}
